mhkit/tests/loads/test_loads.py

```diff
@@ -442,11 +440,9 @@ def test_mler_export_time_series(self):
         mler["WaveSpectrum"] = self.mler["Norm_Spec"].values
         mler["Phase"] = self.mler["phase"].values
         k = resource.wave_number(wave_freq, 70)
-        k = k.fillna(0)
+        np.nan_to_num(k, 0)
```

Comment:
The zero frequency in ``wave_freq`` results in a nan value. The call to ``np.nan_to_num()`` updates the input ``k`` in my testing, which is useful if k is not a numpy array as the input data is both updated and retains its type. If it's more clear for this particular test, I can update to redefine ``k``

Reply:
This is not a limiting factor to getting the PR through but are you saying `np.nan_to_num` modifies k inplace when you run it?

The docs say the function returns the input and that was my experience when I paused the code here:
https://numpy.org/doc/2.0/reference/generated/numpy.nan_to_num.html

Reply:
@ssolson you are correct. I didn't look at the docstring close enough when converting pandas ``fillna`` to numpy ``nan_to_num``. ``np.nan_to_num(k, 0)`` is setting ``copy`` to False and changing k in place, instead of specifying the fill value which already defaults to 0. I'll update this line so that we're using the function correctly